modules/ROOT/pages/errors/gql-errors.adoc

```diff
@@ -0,0 +1,107 @@
+:description: This section describes the GQLSTATUS errors that Neo4j can return, grouped by category, and an example of when they can occur.
+
+[[neo4j-gqlstatus-errors]]
+= List of all GQLSTATUS server error codes
+
+The following page provides an overview of all server errors in Neo4j, along with some scenarios and their possible solutions.
+All errors in Neo4j have severity level `ERROR`.
+
+* <<_connection-exceptions, Connection exceptions>>
+
+
+[[_connection-exceptions]]
+== Connection exceptions
+
+The following is a list of the connection exceptions that can occur in Neo4j.
+
+=== 08000
+
+Notifies the client that an exception occurred during the connection.
+
+.Error details
+[cols="<1s,<4"]
+|===
+|GQLSTATUS code
+m|0800
+|Status description
+a|error: connection exception
+|Classification
+m|CLIENT_ERROR
+|===
+
+=== 08007
+
+Notifies the client that an exception occurred during the resolution of a transaction.
+
+.Error details
+[cols="<1s,<4"]
+|===
+|GQLSTATUS code
+m|08007
+|Status description
+|error: connection exception - transaction resolution unknown.
```

Comment:
```suggestion
|error: connection exception - transaction resolution unknown
```

The dot is a separator between subcondition and message, so if there is no message, there is no dot